tools/launcher/common/eagle3/dump_offline_data_vllm.sh

```
@@ -0,0 +1,55 @@
+    TASK_COUNT=1
+else
+    echo "SLURM_ARRAY_TASK_COUNT ${SLURM_ARRAY_TASK_COUNT}"
+    TASK_COUNT=${SLURM_ARRAY_TASK_COUNT}
+fi
+
+python3 modules/Model-Optimizer/examples/speculative_decoding/collect_hidden_states/compute_hidden_states_vllm.py \
+    --model ${HF_MODEL_CKPT} \
+    --dp-rank ${TASK_ID} \
+    --dp-world-size ${TASK_COUNT} \
+    --trust_remote_code \
+    ${@}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)